modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/client/OpcUaEndpointFilter.java

```diff
@@ -72,9 +72,13 @@ public OpcUaEndpointFilter(@NotNull String configPolicyUri, @NotNull OpcUaAdapte
 
     private EndpointDescription endpointUpdater(EndpointDescription endpoint) {
         if (adapterConfig.getOverrideUri()) {
-            return EndpointUtil.updateUrl(endpoint,
+            final EndpointDescription endpointDescription = EndpointUtil.updateUrl(endpoint,
                     EndpointUtil.getHost(adapterConfig.getUri()),
                     EndpointUtil.getPort(adapterConfig.getUri()));
+            if (log.isInfoEnabled()) {
```

Comment:
either always check for log.isInfoEnabled or never. I'd recommend never, since info is basically default in any logback